contributors/contributors/core-committers.md

```diff
@@ -6,9 +6,9 @@ A core committer is a maintainer on the Mattermost project that has merge access
 
 Below is the list of core committers working on Mattermost:
 
-* [**Angelos Kyratzakos**](core-committers.md)
-  * @angelos.kyratzakos on [community.mattermost.com](https://community.mattermost.com/core/messages/@angelos.kyratzakos) and [@angeloskyratzakos](https://github.com/angeloskyratzakos) on GitHub
-  * Dev areas: Kubernetes, Infrastructure
+* [**Adolfo García Veytia**](core-committers.md)
+  * @puerco on [community.mattermost.com](https://community.mattermost.com/core/messages/@puerco) and [@puerco](https://github.com/puerco) on GitHub
+  * Dev areas: DevOps
```

Comment:
I would suggest adding `SBOM`, `Containers`, `Kubernetes`, `Supply Chain`

anything else you want to add @puerco

Reply:
That is more than enough, thank you!